zerver/tests/tests.py

```diff
@@ -1548,6 +1548,24 @@ def check_for_toggle_param(self, pattern, param):
         user_profile = get_user_profile_by_email("hamlet@zulip.com")
         self.assertEqual(getattr(user_profile, param), False)
 
+    def check_for_toggle_param_patch(self, pattern, param):
```

Comment:
Do we have another method somewhere else that does this?  If so, we should either try to make the original method more reusable or add a TODO comment here saying that we should eventually consolidate the methods.